SourceryRuntime/Sources/Variable.swift

```diff
@@ -51,6 +51,20 @@ public final class Variable: NSObject, SourceryModel, Typed, Annotated {
         return attributes[Attribute.Identifier.final.name] != nil
     }
 
+    /// Reference to type name where the variable is defined,
+    /// nil if defined outside of any `enum`, `struct`, `class` etc
+    public let definedInTypeName: TypeName?
+
+    /// Actual type name where the variable is defined if declaration uses typealias, otherwise just a `definedInTypeName`
+    public var actualDefinedInTypeName: TypeName? {
+        return definedInTypeName?.actualTypeName ?? definedInTypeName
```

Comment:
@ilyapuchka I've checked and yeah for typealiases the `actualDefinedInTypeName` is not resolved into correct value. I have two ideas how to solve it:
1. As we require mainly the information whether the type defined in extension or not, we pass additional parameter to `resolveMethodTypes` or `resolveVariableTypes` whether defined in extension. So when the method is resolving the type it will set it as `isExtension`;
2. Second option is to keep the `definedInType` as we have now, but still call `resolveType` for `definedInTypeName` so it will have `actualDefinedInTypeName` resolved properly.